<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>778</width>
         <height>518</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuComponents">
    <property name="title">
     <string>Components</string>
    </property>
    <addaction name="actionButton"/>
    <addaction name="actionGround"/>
    <addaction name="actionVcc"/>
    <addaction name="separator"/>
    <addaction name="actionLED"/>
    <addaction name="actionDigital_indicator"/>
    <addaction name="separator"/>
    <addaction name="actionOscillator"/>
    <addaction name="actionBeeper"/>
    <addaction name="actionOutputs_extender"/>
    <addaction name="separator"/>
    <addaction name="actionLogic"/>
    <addaction name="actionTrigger"/>
    <addaction name="separator"/>
    <addaction name="actionRegister"/>
    <addaction name="actionDecoder"/>
    <addaction name="actionEncoder"/>
    <addaction name="actionCounter"/>
    <addaction name="actionMultiplexer"/>
    <addaction name="actionAdder"/>
    <addaction name="actionComparator"/>
    <addaction name="actionRandom_generator"/>
    <addaction name="actionRAM"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCompact_layout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuComponents"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarComponents">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionCompact_layout">
   <property name="text">
    <string>Compact layout</string>
   </property>
  </action>
  <action name="actionAdd_button">
   <property name="text">
    <string>Add button</string>
   </property>
  </action>
  <action name="actionAdd_Led">
   <property name="text">
    <string>Add LED</string>
   </property>
  </action>
  <action name="actionButton">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/button.png</normaloff>:/images/button.png</iconset>
   </property>
   <property name="text">
    <string>Button</string>
   </property>
   <property name="whatsThis">
    <string>QCPButton</string>
   </property>
  </action>
  <action name="actionLED">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/led.png</normaloff>:/images/led.png</iconset>
   </property>
   <property name="text">
    <string>LED</string>
   </property>
   <property name="whatsThis">
    <string>QCPLed</string>
   </property>
  </action>
  <action name="actionLogic">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/logic.png</normaloff>:/images/logic.png</iconset>
   </property>
   <property name="text">
    <string>Logic element</string>
   </property>
   <property name="whatsThis">
    <string>QCPLogic</string>
   </property>
  </action>
  <action name="actionOutputs_extender">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/extender.png</normaloff>:/images/extender.png</iconset>
   </property>
   <property name="text">
    <string>Outputs extender</string>
   </property>
   <property name="whatsThis">
    <string>QCPExtender</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="icon">
    <iconset theme="zoom-draw">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom...</string>
   </property>
  </action>
  <action name="actionTrigger">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/trigger.png</normaloff>:/images/trigger.png</iconset>
   </property>
   <property name="text">
    <string>Trigger</string>
   </property>
   <property name="whatsThis">
    <string>QCPTrigger</string>
   </property>
  </action>
  <action name="actionOscillator">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/generator.png</normaloff>:/images/generator.png</iconset>
   </property>
   <property name="text">
    <string>Oscillator</string>
   </property>
   <property name="whatsThis">
    <string>QCPGen</string>
   </property>
  </action>
  <action name="actionAsyncronous_repeater">
   <property name="icon">
    <iconset>
     <normaloff>:/images/repeater.png</normaloff>:/images/repeater.png</iconset>
   </property>
   <property name="text">
    <string>Asyncronous repeater</string>
   </property>
   <property name="whatsThis">
    <string>QCPRepeater</string>
   </property>
  </action>
  <action name="actionBeeper">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/beeper.png</normaloff>:/images/beeper.png</iconset>
   </property>
   <property name="text">
    <string>Beeper</string>
   </property>
   <property name="whatsThis">
    <string>QCPBeeper</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/register.png</normaloff>:/images/register.png</iconset>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
   <property name="whatsThis">
    <string>QCPRegister</string>
   </property>
  </action>
  <action name="actionDecoder">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/decoder.png</normaloff>:/images/decoder.png</iconset>
   </property>
   <property name="text">
    <string>Decoder</string>
   </property>
   <property name="whatsThis">
    <string>QCPDecoder</string>
   </property>
  </action>
  <action name="actionEncoder">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/encoder.png</normaloff>:/images/encoder.png</iconset>
   </property>
   <property name="text">
    <string>Encoder</string>
   </property>
   <property name="whatsThis">
    <string>QCPEncoder</string>
   </property>
  </action>
  <action name="actionCounter">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/counter.png</normaloff>:/images/counter.png</iconset>
   </property>
   <property name="text">
    <string>Counter</string>
   </property>
   <property name="whatsThis">
    <string>QCPCounter</string>
   </property>
  </action>
  <action name="actionDigital_indicator">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/digit.png</normaloff>:/images/digit.png</iconset>
   </property>
   <property name="text">
    <string>Digital indicator</string>
   </property>
   <property name="whatsThis">
    <string>QCPDigit</string>
   </property>
  </action>
  <action name="actionMux">
   <property name="text">
    <string>Mux</string>
   </property>
  </action>
  <action name="actionMultiplexer">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/mux.png</normaloff>:/images/mux.png</iconset>
   </property>
   <property name="text">
    <string>Multiplexer</string>
   </property>
   <property name="whatsThis">
    <string>QCPMux</string>
   </property>
  </action>
  <action name="actionAdder">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/sum.png</normaloff>:/images/sum.png</iconset>
   </property>
   <property name="text">
    <string>Adder</string>
   </property>
   <property name="whatsThis">
    <string>QCPSum</string>
   </property>
  </action>
  <action name="actionGround">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/gnd.png</normaloff>:/images/gnd.png</iconset>
   </property>
   <property name="text">
    <string>Ground</string>
   </property>
   <property name="whatsThis">
    <string>QCPGnd</string>
   </property>
  </action>
  <action name="actionVcc">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/vcc.png</normaloff>:/images/vcc.png</iconset>
   </property>
   <property name="text">
    <string>Vcc</string>
   </property>
   <property name="whatsThis">
    <string>QCPVcc</string>
   </property>
  </action>
  <action name="actionComparator">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/comparator.png</normaloff>:/images/comparator.png</iconset>
   </property>
   <property name="text">
    <string>Comparator</string>
   </property>
   <property name="whatsThis">
    <string>QCPComparator</string>
   </property>
  </action>
  <action name="actionRandom_generator">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/rnd.png</normaloff>:/images/rnd.png</iconset>
   </property>
   <property name="text">
    <string>Random generator</string>
   </property>
   <property name="whatsThis">
    <string>QCPRndGen</string>
   </property>
  </action>
  <action name="actionRAM">
   <property name="icon">
    <iconset resource="exsim.qrc">
     <normaloff>:/images/ram.png</normaloff>:/images/ram.png</iconset>
   </property>
   <property name="text">
    <string>RAM</string>
   </property>
   <property name="whatsThis">
    <string>QCPRAM</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="exsim.qrc"/>
 </resources>
 <connections/>
</ui>
